pkg/events/event_store.go

```diff
@@ -43,8 +43,9 @@ type EventStore struct {
 
 func newEventStore(size uint64) *EventStore {
 	return &EventStore{
-		events: make([]*si.EventRecord, size),
-		size:   size,
+		events:   make([]*si.EventRecord, size),
+		size:     size,
+		lastSize: size,
```

Comment:
Set initial lastSize value to prevent uncessarily resize in  https://github.com/apache/yunikorn-core/blob/634380f109a0ad724cabcbdb5e2e7d78ec2debd5/pkg/events/event_store.go#L70-L73

Reply:
that is a good finding. Could you add UT for that?

Reply:
Done. 74b9c1ee4e8ba7638e62c2c1b4647940419ac18d
Extend the existing test: TestSetStoreSize()